doc/developers/contributing.rst

```diff
@@ -207,6 +207,51 @@ and Cython optimizations.
    <http://astropy.readthedocs.org/en/latest/development/workflow/development_workflow.html>`_
    sections.
 
+Rebasing and squashing commits
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Thanks for checking. Maybe it would be better to have a heading like [this](http://scikit-learn.org/stable/developers/contributing.html#input-validation)?